src/main/webapp/resources/js/components/samples/components/SequencingObjectList.tsx

```diff
@@ -34,32 +34,46 @@ import {
   fetchUpdatedSequencingObjects,
   useUpdateDefaultSampleSequencingObjectMutation,
 } from "../../../apis/samples/samples";
+import { CheckboxChangeEvent } from "antd/lib/checkbox";
+import { SampleSequencingObject, SequencingObject } from "../../../types/irida";
 
-const fileProcessTranslations = {
+const fileProcessTranslations: { [key: string]: any } = {
   UNPROCESSED: i18n("SampleFilesList.fileProcessingState.UNPROCESSED"),
   QUEUED: i18n("SampleFilesList.fileProcessingState.QUEUED"),
   PROCESSING: i18n("SampleFilesList.fileProcessingState.PROCESSING"),
   FINISHED: i18n("SampleFilesList.fileProcessingState.FINISHED"),
   ERROR: i18n("SampleFilesList.fileProcessingState.ERROR"),
 };
 
+export interface SequencingObjectListProps {
+  removeSampleFiles: ({
+    fileObjectId,
+    type,
+  }: {
+    fileObjectId: number;
+    type: string;
+  }) => void;
+}
+
 /**
  * React component to display, remove, download sequencing objects
  * @param {function} removeSampleFiles The function to remove sequencing objects
  * @returns {JSX.Element}
  * @constructor
  */
-export function SequencingObjectList({ removeSampleFiles = () => {} }) {
+export function SequencingObjectList({
+  removeSampleFiles = () => {},
+}: SequencingObjectListProps): JSX.Element {
   const [updateSampleDefaultSequencingObject] =
     useUpdateDefaultSampleSequencingObjectMutation();
 
   const {
     sample,
     modifiable: isModifiable,
     projectId,
-  } = useSelector((state) => state.sampleReducer);
+  } = useSelector((state: RootStateOrAny) => state.sampleReducer);
   const { files, concatenateSelected } = useSelector(
-    (state) => state.sampleFilesReducer
+    (state: RootStateOrAny) => state.sampleFilesReducer
```

Comment:
Update

Reply:
Updated in 445c58cec70fbfbbe6113426d30eb0caf860e90f